### DISCUSSION TITLE
Spawn Enemy C# API

### DISCUSSION THREAD

**Discussion by Eugeny335** [praydog/REFramework#1110] [General Questions] [answered]
Can I use the API for the c# plugin to create enemies and, if possible, is it possible to give them a name above their head?

**Comment by praydog** [comment, marked as answer]
Probably. Look at how EMV engine does it. It will probably involve folders or prefab instantiation.

You can use imgui and via.math.worldPos2ScreenPos to draw above their head.

**Reply by Eugeny335** [reply]
Can you tell me the path to Prefabs folder in RE4?
